docs/table-of-contents.json

```diff
@@ -214,27 +210,27 @@
           "entries": [
             {
               "name": "Cone",
```

Comment:
The names should be `ConeGeometry`, `CubeGeometry`, etc.

Reply:
Testing this out, it seems like this "name" field is actually ignored entirely and that the title is dynamically pulled out of the header of the page in markdown.  The doc markdown for these geometry classes needs to be updated to match the new class names.  Will address in a separate PR.

Reply:
Yes the "name" field is a relic of the old ocular (pre-gatsby) config. gatsby only uses "entry" at the moment.